.github/workflows/pull-request.yml

```
@@ -0,0 +1,1162 @@
+        run: pnpm fetch
+      - name: Install Dependencies
+        run: pnpm install --frozen-lockfile --prefer-offline
+
+      - name: Count number of lines
+        run: |
+          chmod +x ./.github/workflows/scripts/countline.py
+          ./.github/workflows/scripts/countline.py --lines 600 --exclude_files src/screens/LoginPage/LoginPage.tsx src/GraphQl/Queries/Queries.ts src/screens/OrgList/OrgList.tsx src/GraphQl/Mutations/mutations.ts src/components/EventListCard/EventListCardModals.tsx src/components/TagActions/TagActionsMocks.ts src/utils/interfaces.ts src/screens/MemberDetail/MemberDetail.tsx src/components/OrgPostCard/OrgPostCard.tsx src/components/UsersTableItem/UsersTableItem.tsx src/components/UserPortal/ChatRoom/ChatRoom.tsx
+
+      - name: Get changed TypeScript files
+        id: changed-files
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:10:60: Double quote to prevent globbing and word splitting

(shellcheck)